userguide/content/en/docs/adding-content/shortcodes/index.md

```diff
@@ -706,4 +706,50 @@ This text does not appear in the output if `buildCondition = "bar" is set in you
+**Abstraction** is the process of showing only essential/necessary features of an entity/object to the outside world and hide the other irrelevant information. For example to shut on/off your TV you only have a on/off method (≙ power button). It is not required to understand what internal processes are triggered in order to power on/off your TV device.
+{{% /accordion-item %}}
+{{% accordion-item header="Principle 2:&nbsp;**Encapsulation**" %}}
+**Encapsulation** means wrapping up data and member functions (methods) together into a single unit, commonly referred to as class. One aspect of encapsulation is the concept of data hiding. We can hide data by marking variables as private and expose only needed properties (getters and setters) to the public. Internally, these properties make use of the private data not exposed to the public.
+{{% /accordion-item %}}
+{{% accordion-item header="Principle 3:&nbsp;**Inheritance**" %}}
+**Inheritance** is the ability of creating a new class from an existing class. A child object acquires the property of a parent object. By means of inheritance, we achieve that a child class (subclass) acquires the properties and behavior of another parent class (super-class).
+{{% /accordion-item %}}
+{{% accordion-item header="Principle 4:&nbsp;**Polymorphism**" %}}
+**Polymorphism** is derived from two Greek words: _πολύς_ (`polús`) translates to `many` and _μορφή_ (`morphḗ`) to `form`. So polymorphism means `many forms`. A child/subclass can define its own unique behavior and still share the same functionalities or behavior of its parent/base class. A subclass can have their own behavior and share some of its behavior from its parent class not the other way around. A parent class cannot have the behavior of its subclass.
+{{% /accordion-item %}}
+{{% /accordion %}}
+
+The Docsy template provides two shortcodes `accordion` and `accordion-item` that let you easily create an accordion component. To see how to use them, have a look at the following code block, which renders to the accordion shown above:
+
+```go-html-template
+{{%/* accordion alwaysOpen=false */%}}
+{{%/* accordion-item header="Principle 1: Abstraction" */%}}
+**Abstraction** is the process of ...
+{{%/* /accordion-item */%}}
+{{%/* accordion-item header="Principle 2: Encapsulation" */%}}
+**Encapsulation** means wrapping up data ...
+{{%/* /accordion-item */%}}
+{{%/* accordion-item Principle 3: Inheritance */%}}
+**Inheritance** is the ability of creating a new class ...
+{{%/* /accordion-item */%}}
+... fourth item omitted
+{{%/* /accordion %}}
+```
+
+### Shortcode details
+
+An accordion component is implemented using two shortcodes:
+
+* The `accordion` shortcode, which is the container element for the collapsible items of the accordion. This shortcode can hold the optional named parameters `alwaysOpen`, and `flush`. Specify `alwaysOpen=true` if you want to make accordion items stay open even when another item is opened.
```

Comment:
```suggestion
* The `accordion` shortcode, which is the container element for the collapsible items of the accordion. This shortcode can hold the optional named parameters `alwaysOpen` and `flush`. Specify `alwaysOpen=true` if you want to make accordion items stay open even when another item is opened.
```

Reply:
Removed the superfluous comma, thanks for spotting this!